src/main/java/com/artipie/asto/misc/UncheckedIOConsumer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie.asto.misc;
+
+import com.artipie.asto.ArtipieIOException;
+import java.io.IOException;
+import java.util.function.Consumer;
+
+/**
+ * Unchecked IO {@link Consumer}.
+ * @param <T> Consumer type
+ * @since 1.1
+ * @checkstyle AbbreviationAsWordInNameCheck (200 lines)
+ */
+public final class UncheckedIOConsumer<T> implements Consumer<T> {
+
+    /**
+     * Checked version.
+     */
+    private final UncheckedConsumer.Checked<T, IOException> checked;
```

Comment:
Here we can use `UncheckedConsumer.Checked<T, ? extends IOException>` to allow IO consumers to throw `FileNotFoundException` or other `IOException` subtypes (`Checked<FileNotFoundException>` as param)